gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/v4/entrypoint/EntrypointResource.java

```diff
@@ -112,4 +112,26 @@ public String getEntrypointDoc(@PathParam("entrypoint") String entrypoint) {
 
         return entrypointService.getDocumentation(entrypoint);
     }
+
+    @GET
+    @Path("subscriptionSchema")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "🧪 Get a entrypoint's subscription schema",
+        description = "⚠️ This resource is in alpha version. This implies that it is likely to be modified or even removed in future versions. ⚠️. <br><br>User must have the ENVIRONMENT_API[READ] permission to use this service"
+    )
+    @ApiResponse(
+        responseCode = "200",
+        description = "Entrypoint schema",
+        content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = String.class))
+    )
+    @ApiResponse(responseCode = "404", description = "Entrypoint not found")
```

Comment:
Should we also return 404 if there is no subscription schema for this entrypoint ?
I assume actually it will return a 200 with null content

Reply:
To stay consistent with other resources of this class I keep the behaviour: if no schema-form, then a `204 No Content `is sent.

I added some `@ApiResponse(responseCode = "204", description = "Subscription schema not found")` to improve documentation